daml/ContingentClaims/Claim.daml

```diff
@@ -114,29 +114,30 @@ isBearerExecutor = cata \case
 
 -- | Converts a `Serialized.Claim` read from ledger, to a more general `Claim`, for use with `Daml.Control.Recursion`.
 deserialize : Serialized.Claim t a -> Claim Observation t a
-deserialize = ana \case
+deserialize = futu \case
```

Comment:
What does the change from `ana` to `futu` bring ?

Reply:
Hey, I added some doc to daml-ctl, so hopefully you can see some comments for these functions in your IDE now.

a `futumorphism` allows you to unfold more than a single level at a time. The first parameter is normally a `a -> f a` for `cata`, so there's no way to unfold more than a single `AndF` in one go (e.g. two levels would be `a -> f (f a)`); the type system prevents you from doing this. `futu` by contrast allows you to unfold multiple levels of the tree, safely.